EIPS/Muir Glacier upgrade postmortem.md

```diff
@@ -0,0 +1,127 @@
+---
+eip: TBD
+title: Muir Glacier upgrade postmortem report
+author: James Hancock and Pooja Ranjan 
```

Comment:
FYI, this formatting for authors is invalid.

Reply:
```suggestion
author: James Hancock, Pooja Ranjan 
```

Reply:
```suggestion
author: James Hancock (@MadeofTin), Pooja Ranjan (@poojaranjan)
```

Reply:
I believe either an email address or GitHub handle is required @lightclient, not just names.

Reply:
"Theoretically" EIP-1 allows for only names in the author field. Not saying that's good though!

Reply:
TIL that handle/email is not required.  :)

Reply:
That's a good point.

Reply:
@lightclient @MicahZoltu I've incorporated suggested changes yet 'All checks have failed', what am I missing?
Due to ongoing discussion of where this informational EIP should be stored, I understand it can't be merged just yet, but wondering about the correct format to be able to move beyond the bot check.